signwriting_animation/diffusion/tests/test_overfitting.py

```diff
@@ -0,0 +1,118 @@
+import torch
```

Comment:
this test file tests for 2 things at once:
input of (X signwriting and Y history should return Y)
input of (A signwriting and B history should return B)

Now, how can we tell that both X and Y play a role here? as in, what if the model completely ignores the signwriting and just looks at the history? or the other way around?

For this, I recommend now using 4 examples:
```
("M518x529S14c20481x471S27106503x489", 0, -1),        
("M518x529S14c20481x471S27106503x489", 1, 1),        
("M518x533S1870a489x515S18701482x490", 0, 0),
("M518x533S1870a489x515S18701482x490", 1, 2),
 ```

This way, we can tell for sure that both variables play a role in the prediction